### DISCUSSION TITLE
Onboarding

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#19199] [Feature Requests]
### Summary

Is there an extensions from which it would be possible to implement onboarding?

### Basic Example

https://v-onboarding.fatihsolhan.com/

### Motivation

On complex forms (e.g. 3 accordions of 5 groups each with inputs inside) it would be nice to have onboarding explaining the role of tha accordions and groups.

### Detailed Design

There is may be already an extension that makes it easy to implement Onboarding?

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

### Unresolved Questions

Tried to do this from an input interface (creating a dummy field), but, although no error, it does not work: nothing shown.
From https://v-onboarding-docs.fatihsolhan.com/ I created the following interface
index.js
```
import InterfaceComponent from './interface.vue';

export default {
	id: 'onboarding',
	name: 'Onboarding',
	icon: 'help',
	description: 'Interactive Help',
	component: InterfaceComponent,
	options: null,
	types: ['string'],
};

```
interface.vue
```
<template>
	<VOnboardingWrapper ref="wrapper" :steps="steps" />
	<div>
		<button id="onboarding">Welcome</button>
	</div>
</template>

<script>
import { defineComponent, ref, onMounted } from 'vue';
import { VOnboardingWrapper, useVOnboarding } from 'v-onboarding';
import './style.css';

export default defineComponent({
	components: {
		VOnboardingWrapper
	},

	setup() {
		const wrapper = ref(null);
		const onboarding = useVOnboarding(wrapper);
  		const steps = [
			{ attachTo: { element: "input[field='network']" }, content: { title: "Network..." } },
			{ attachTo: { element: "input[field='project']" }, content: { title: "Acronym..." } }
		];

		onMounted(async () => {
			console.log('onboarding.start')
			debugger
			onboarding.start();

			return { wrapper, ...onboarding, steps };
		});
	},
});
</script>

```
But it looks like it does not start.

There is also https://github.com/shipshapecode/vue-shepherd#readme (https://www.npmjs.com/package/vue-shepherd)

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request. For questions and discussions, [Discord](https://directus.chat) is the better place 👍🏻
